IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/WolfSSLDemo/src/wolfDemo/wolf_main.c

```diff
@@ -0,0 +1,265 @@
+/*wolf_main.c
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+#include <string.h>
+#include <wolf_main.h>
+
+#include "r_sys_time_rx_if.h"
+#include "r_cmt_rx_if.h"
+#include "Pin.h"
+#include "r_t4_itcpip.h"
+#define SSL_SERVER
+#define ETHER_TASK
+extern const UB _t4_dhcp_enable;
+static int dhcp_accept_flg = 0;
+static UW tcpudp_work[14800] = {0};
+UB	dhcp_evt = 0;
+void dhcp_check(void);
+void print_dhcp(VP param);
+#define DHCP_ENABEL        (1)
+#define IF_CH_NUMBER       (0)
+
+void main(void)
+{
+/*Toopers start function */
```

Comment:
Fix indentation in file. Should be 4 spaces. Mess all the way through.

Reply:
Fixed and committed.